lib/aida/json_schema.ex

```diff
@@ -72,6 +72,8 @@ defmodule Aida.JsonSchema do
   defp get_validation_errors(object, type, schema) do
     type_string = type |> to_string
     type_schema = schema.schema["definitions"][type_string]
+    require Logger
```

Comment:
(If this change was intended) it maybe better to move the require statement to the top of the file.